pkg/store/azure_keyvault_store.go

```
@@ -0,0 +1,208 @@
+	stackParts := strings.Split(stack, *s.stackDelimiter)
+	componentParts := strings.Split(component, "/")
+
+	parts := append([]string{s.prefix}, stackParts...)
+	parts = append(parts, componentParts...)
+	parts = append(parts, key)
+
+	// Azure Key Vault secret names can only contain alphanumeric characters and dashes
+	secretName := strings.ToLower(strings.Join(parts, "-"))
+	secretName = strings.Trim(secretName, "-")
+
+	if len(secretName) > 127 {
```

golangci-lint (1.64.8)

[error] add-constant: avoid magic numbers like '127', create a named constant for it

(revive)